src/internal/queries/getNumberOfDelegatorsQuery.graphql

```diff
@@ -0,0 +1,8 @@
+query getNumberOfDelegators($poolAddress: String) {
```

Comment:
nit
this and getNumberOfDelegatorsForAllPools can be one query that accept optional where condition, and then the function can pass in the where, something like https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/providers/indexer.ts#L306